### ISSUE TITLE
Bug: CPU Core and Voltage Incorrect After Enabling Core Isolation in Windows 11

### ISSUE THREAD

**Issue by schiffy91** [ArcadeRenegade/SidebarDiagnostics#405]
After enabling Core Isolation on Windows 11, SideBarDiagnostics reports my Ryzen 5950x has a frequency of 15GhZ -> NaN frequently. It also reports my CPU voltage now going up scary high to something like 1.55V. Opening up Ryzen Master and the latest version of HwInfo show this is isn't right. SideBarDiagnostics may need to be compiled with a new version of HwInfo

**Comment by ArcadeRenegade** [comment]
Does this issue happen with the latest commit of Libre Hardware Monitor?

**Comment by schiffy91** [comment]
Good catch. It does as per the latest stable version released (0.9). I filed a bug there: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/801. Should I leave this open, or close it until it's been fixed in LibreHardwareMonitor?

**Comment by ArcadeRenegade** [comment]
You can leave this open, I need to do a new release soon anyways.

**Comment by Nostrildumbass** [comment]
I believe the app is displaying VID as opposed to actual voltage (VCORE) the motherboard is supplying. Is this by design? I would suggest adding the option for which to display in the Monitor config

**Comment by brunoherrera** [comment]
this was never fixed, I have same hardware 5950x
